src/core/lib/channel/metrics.h

```diff
@@ -231,6 +327,37 @@ class GlobalStatsPluginRegistry {
       ABSL_GUARDED_BY(mutex_);
 };
 
+// A metric callback that is registered with a stats plugin group.
+class RegisteredMetricCallback {
+ public:
+  RegisteredMetricCallback(
```

Comment:
do we want each component to have to reason about a reasonable `min_interval`? 

How about another constructor which sets the interval to a default that we think will work for most metrics?

Reply:
This won't actually be instantiated directly by gRPC components; instead, they will use `StatsPluginGroup::RegisterCallback()` to create this object, and that method does have a default value for this parameter.